### PR TITLE
feat: Mun benefits Section (Home Page)

### PR DESCRIPTION
**Pull request by Rajani169**

### CODE REVIEW

**Comment by amritsrivastava** [comment]
- Include the benefits section in about `div` in the index file
- Also, reduce the top margin of the about section a Lil bit so that there will not be lots of empty space.

File: src/pages/index.js [hunk 1]
```diff
@@ -116,9 +117,18 @@ export default () => {
           />
         </div>
       </Banner>
-      <div id='about' style={{ minHeight: '100vh' }}>
```
Comment (amritsrivastava) [inline src/pages/index.js:119]: Why you removed `minHeight`

File: src/pages/index.js [hunk 2]
```diff
@@ -116,9 +117,18 @@ export default () => {
...
+      <div id='about'>
         <About />
+        <div
+          id='why-mun'
```
Comment (amritsrivastava) [inline src/pages/index.js]: What is the need of assigning this id ??

File: src/pages/index.js [hunk 3]
```diff
@@ -116,9 +117,18 @@ export default () => {
...
+          style={{
+            background: theme.palette.background.pinkish,
+          }}
```
Comment (amritsrivastava) [inline src/pages/index.js]: apply this styling to outer div of about section

**Review by amritsrivastava — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (Rajani169) [reply src/pages/index.js:119]: it just happened accidentally. placed it back!

File: src/pages/index.js (hunk 3 above)
Comment (Rajani169) [reply src/pages/index.js]: done

File: src/pages/index.js (hunk 2 above)
Comment (Rajani169) [reply src/pages/index.js]: removed this.

File: src/components/sections/home/whyMun.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        src='/images/line.png'
+        className='mx-auto pb-5'
+        alt='---------------------'
+      />
+      <Grid container justify='center'>
+        <Grid item md={4}>
+          <div
+            className={`my-10 mx-10 justify-center text-center flex flex-col ${classes.containerContent}`}
+          >
+            <div className='flex justify-center'>
```
Comment (amritsrivastava) [inline src/components/sections/home/whyMun.js]: @Rajani169 why you need these two extra divisions ??

File: src/components/sections/home/whyMun.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+              <p
+                className={`rounded-full h-32 w-32 bg-white flex items-center justify-center ${classes.borderPink}`}
+              >
+                <CastForEducationIcon style={{ fontSize: '50px' }} />
+              </p>
```
Comment (amritsrivastava) [inline src/components/sections/home/whyMun.js]: Explain the purpose of this paragraph tag for making a round circle out of icon.
Can't you use `div` here ?

File: src/components/sections/home/whyMun.js [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+            </div>
+            <div className='w-full my-5 text-black'>
```
Comment (amritsrivastava) [inline src/components/sections/home/whyMun.js]: Why you are using `w-full` and `text-black` class??

File: src/components/sections/home/whyMun.js [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+              <Typography variant='h5'>Skills</Typography>
+            </div>
+            <div className='text-black'>
```
Comment (amritsrivastava) [inline src/components/sections/home/whyMun.js]: again using `text-black` ?

**Review by amritsrivastava — CHANGES_REQUESTED** [review]

**Comment by amritsrivastava** [comment]
Also, fix this styling for every card.

File: src/components/sections/home/whyMun.js (hunk 5 above)
Comment (Rajani169) [reply src/components/sections/home/whyMun.js]: for hovering effect

File: src/components/sections/home/whyMun.js (hunk 6 above)
Comment (Rajani169) [reply src/components/sections/home/whyMun.js]: actually for using hovering effect over that grid item, i had to change the color to primary due to which the text color of the paragraph also got effected and I applied that "text-black" class.

File: src/components/sections/home/whyMun.js (hunk 7 above)
Comment (Rajani169) [reply src/components/sections/home/whyMun.js]: same reason

File: src/components/sections/home/whyMun.js (hunk 4 above)
Comment (Rajani169) [reply src/components/sections/home/whyMun.js]: outer div is for the whole card and inner div is for the circlular icons

**Pull request merged**